README.md

```diff
@@ -1987,4 +1987,60 @@ and a your customjsonform fragment class initialize this interactor from createP
         .....
      }
  ````
+ 
+ ### MultiSelectList Widget
```

Comment:
This is nice, if you were to use this the first time, would this information just get you started on it? 

Kindly see if you schedule time to add it to the native form sample which makes it easier to understand how the widget works and looks like

Reply:
sample is there